scipy/fftpack/realtransforms.py

```diff
@@ -581,60 +588,65 @@ def dst(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False):
 
     **Type I**
 
-    There are several definitions of the DST-I; we use the following
-    for ``norm=None``.  DST-I assumes the input is odd around n=-1 and n=N. ::
+    There are several definitions of the DST-I; we use the following for
+    ``norm=None``. DST-I assumes the input is odd around :math:`n=-1` and
+    :math:`n=N`.
+
+    .. math::
 
-                 N-1
-      y[k] = 2 * sum x[n]*sin(pi*(k+1)*(n+1)/(N+1))
-                 n=0
+        y_k = 2 \sum_{n=0}^{N-1} x_n \sin\left(\frac{\pi(k+1)(n+1)}{N+1}\right)
 
-    Note that the DST-I is only supported for input size > 1
-    The (unnormalized) DST-I is its own inverse, up to a factor `2(N+1)`.
-    The orthonormalized DST-I is exactly its own inverse.
+    Note that the DST-I is only supported for input size > 1. The
+    (unnormalized) DST-I is its own inverse, up to a factor :math:`2(N+1)`. The
+    orthonormalized DST-I is exactly its own inverse.
 
     **Type II**
 
-    There are several definitions of the DST-II; we use the following
-    for ``norm=None``.  DST-II assumes the input is odd around n=-1/2 and
-    n=N-1/2; the output is odd around k=-1 and even around k=N-1 ::
+    There are several definitions of the DST-II; we use the following for
+    ``norm=None``. DST-II assumes the input is odd around :math:`n=-1/2` and
+    :math:`n=N-1/2`; the output is odd around :math:`k=-1` and even around
+    :math:`k=N-1`
+
+    .. math::
 
-                N-1
-      y[k] = 2* sum x[n]*sin(pi*(k+1)*(n+0.5)/N), 0 <= k < N.
-                n=0
+        y_k = 2 \sum_{n=0}^{N-1} x_n \sin\left(\frac{\pi(k+1)(2n+1)}{2N}\right)
 
-    if ``norm='ortho'``, ``y[k]`` is multiplied by a scaling factor `f` ::
+    if ``norm='ortho'``, :math:`y_k` is multiplied by a scaling factor :math:`f`
```

Comment:
@rgommers 
>What I would leave out is things like changing `2N` to `:math:2N`

So here, for example, would you leave this line as it was? Even though the 'f' in the text and in the equation will be formatted slightly differently.

Reply:
Yes, I'd probasbly leave that as is.